src/stories/Buttons.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { Button } from "@storybook/react/demo";
+import Moto from "../icons/moto";
+import Profile from "../icons/profile";
+import Chat from "../icons/chat";
+import Showoffers from "../button/showoffers";
+import Tritcher from "../icons/tritcher";
+import Contactowner from "../button/contactowner";
+import Star from "../icons/star";
+
```

Comment:
Maybe you should consider using one story for Icons and the other for Buttons.

Reply:
Yes, it is resolve